system-addon/lib/TelemetryFeed.jsm

```diff
@@ -266,6 +265,13 @@ this.TelemetryFeed = class TelemetryFeed {
     // get blows up.
     let session = this.sessions.get(port);
 
+    // Partial workaround for #3118; avoids the worst incorrect associations of
+    // times with browsers, and should result in no longer having apparent load
+    // times of 500,000 ms.  Better fix for more cases forthcoming.
```

Comment:
The 500sec comment doesn't really make any sense by itself. Probably better to just say to associate the load trigger with the visibility event as the user is most likely associating the trigger to the tab just shown. Could maybe note that this is to not associate with a preloaded browser as those don't get the event until shown.